tools/build-webview

```diff
@@ -138,17 +138,11 @@ fi  # if $sanity_checks
 # (Build and) sync
 ################################################################################
 
-# Set $staging. This is our local staging directory; once it's compiled, it will
-# be synced over to $dest.
-readonly staging="$root/src/webview/static"
-if [ ! -d "$staging" ]; then
-    err "cannot find asset staging directory '$staging'";
-fi
-
-
 # Sync the directory structure, preserving metadata. (We ignore any files named
 # 'README.md'; these should be strictly informative.)
 #
 # We use `rsync` here, as it will skip unchanged files.
 mkdir -p "$dest"
-rsync -a --no-D --delete --delete-excluded --exclude=README.md "$staging/." "$dest"
+rsync -a --no-D --delete --delete-excluded \
```

Comment:
The staging directory was never intended as a speed optimization; the first two paragraphs in this commit are attacking a strawman.

(It was primarily intended for transparency and verifiability, as part of a check against stale files being left in the build directories. The details are probably no longer relevant enough to discuss here.)

Reply:
Interesting -- that reason is news to me :slightly_smiling_face: I guess the sharing was the best explanation I could reverse-engineer for it -- as the message said, there are circumstances not *too* different from these where something like it would be helpful.

Revised to:

    This "staging" step was intended as a way to help the developer
    hand-inspect the output for stale files.  Better to prevent them
    automatically, by continuing to use tools like `rsync --delete`.
    Meanwhile, the extra indirection complicates the code and
    complicates reasoning about it.  So, simplify it away.